frontend/src/stories/TeamsTable.stories.js

```diff
@@ -11,5 +11,6 @@ const Template = args => <TeamsTable {...args} />
 
 export const Primary = Template.bind({});
 Primary.args = {
-  teams: user.teamMemberships,
+  teams: authUser.teamMemberships,
```

Comment:
The authuser and the user are not necessarily the exact same person. You Kaleem, might be looking at Babalwa's board. That would mean that the authuser is you, Kaleem. And the user is Babalwa.